docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -610,10 +638,13 @@ edit:
 delete buttons: False  
 allow reordering: True  
 ---
+code: |
+  questions[i].field_list.there_are_any = questions[i].field_list.number() > 0
+---
```

Comment:
What would happen without this code? Seems related to the comment below, but it's a bit confusing because we aren't gathering anything in the question[i].field_list, it's already there.

Reply:
Maybe it would work now. Before I was getting an error that `.there_are_any` wasn't defined for the question. I can try removing it.

Reply:
Just confirmed that when press 'continue' after selecting my last field item, I get `There was a reference to a variable 'questions[0].field_list.there_are_any' that could not be looked up...`

Reply:
That's fair, just wanted to see. Thanks for checking!